### ISSUE TITLE
Installing lightdm elementary-greeter changing DE to gnome again

### ISSUE THREAD

**Issue by LightColoredLines** [decathorpe/elementary-nightly-rpms#48]
I do install lightdm elementary-greeter light-locker with elementary-nightly repo. And enabling the lightdm service. Also the greeter is fine and working but after login, the environment is not Pantheon anymore. it's Gnome (even after i select Pantheon in login session). But when i enable GDM service again, it came back to Pantheon de.

**Comment by decathorpe** [comment]
This is strange, as I'm actively testing this exact case regularly, and it worked as of yesterday. Can you look at the log output when the session falls back to gnome? it should contain some hints.

PS: Why did you opt for the nightly packages?

**Comment by LightColoredLines** [comment]
> This is strange, as I'm actively testing this exact case regularly, and it worked as of yesterday. Can you look at the log output when the session falls back to gnome? it should contain some hints.
> 
> PS: Why did you opt for the nightly packages?

i've tried using staging copr but same that's why i'm trying nightly repo.
here is the SS of my current situation. neofetch says i'm on Pantheon DE but actua
![Screenshot from 2020-03-28 09-14-10](https://user-images.githubusercontent.com/10070295/77812740-f2ad2580-70d5-11ea-914d-29e263634b6e.png)
lly not.

**Comment by decathorpe** [comment]
Why would you need the staging (or nightly) COPR to begin with? Almost everything is already available from the official fedora repositories, and that's also what I'm testing regularly ... The staging and nightly repos are only there for people who want to do actual testing, not for regular users. That's also what it says on the COPR descriptions :(

Well, gala doesn't appear in your logs at all. So something is definitely wrong. It looks like the GNOME fallback session is started instead of Pantheon, but I can't see anything in the logs that would indicate why.

How did you enable LightDM? Did you also disable GDM? The commands should be

- `systemctl disable gdm.service` and
- `systemctl enable lightdm.service` and reboot.

**Comment by LightColoredLines** [comment]
> * `systemctl disable gdm.service` and
> * `systemctl enable lightdm.service` and reboot.

yes i'm doing that.
actually i'm just curious about it not as a normal user :)
Fedora have a good performance and i want Pantheon as my daily DE.

**Comment by decathorpe** [comment]
No, removing gnome-session will remove Pantheon as well, since Pantheon relies on gnome-session as an underlying technology.

So I'm not sure what's going wrong with your setup.

After you tried to log into Pantheon (and got to GNOME), can you run `journalctl -xb --user | grep gala`,  `journalctl -xb --user | grep gnome-session` and `journalctl -xb --user | grep pantheon`? Those should give some information about why GNOME was started instead of Pantheon.

**Comment by LightColoredLines** [comment]
> No, removing gnome-session will remove Pantheon as well, since Pantheon relies on gnome-session as an underlying technology.
> 
> So I'm not sure what's going wrong with your setup.
> 
> After you tried to log into Pantheon (and got to GNOME), can you run `journalctl -xb --user | grep gala`, `journalctl -xb --user | grep gnome-session` and `journalctl -xb --user | grep pantheon`? Those should give some information about why GNOME was started instead of Pantheon.

here is the log
https://pastebin.com/iynpGg25

**Comment by decathorpe** [comment]
Looks like this error occurs because Pantheon / gala doesn't work without OpenGL acceleration. Looking at the original screenshot, I see you're using VirtualBox ... maybe that's the problem.

**Comment by LightColoredLines** [comment]
> Looks like this error occurs because Pantheon / gala doesn't work without OpenGL acceleration. Looking at the original screenshot, I see you're using VirtualBox ... maybe that's the problem.

i see. so i can't use it on VirtualBox. i'll try installing it on my HDD then.

**Comment by decathorpe** [comment]
I'm not sure, but at least some components seem to be complaining about a lack of GPU acceleration.

I'm using gnome-boxes for my virtual machines, and it works fine there.
